### DISCUSSION TITLE
Can this work with IntelliTouch?

### DISCUSSION THREAD

**Discussion by toddsay** [jlvaillant/intellicenter#2] [Q&A] [answered]
I'm using the older IntelliTouch system, and wondering if this integration can also work with that system. I've installed the integration through HACS but nothing appears in the main integrations screen through discovery. I should mention that my IoT devices are on a different subnet, so automatic discovery tends to be hit-or-miss. Is there a manual yaml configuration I can try, to specify the IP address of my ScreenLogic controller?

**Comment by jlvaillant** [comment, marked as answer]
You can definitely set the IP manually (by adding the intégration you will go thru a standard config dialog) but I don’t think it will work with the IntelliTouch. I believe I looked at it a while ago and the protocol, if i remember, was totally different. Sorry 😞

**Reply by toddsay** [reply]
Thanks for that reply. You're right, I somehow missed a step. I'm now able to enter the IP but it was unable to connect (as you expected).
